packages/ng/forms/multilanguage-input/multilanguage-input.component.ts

```diff
@@ -46,6 +46,10 @@ export class MultilanguageInputComponent implements ControlValueAccessor {
 
 	intl = getIntl(LU_MULTILANGUAGE_INPUT_TRANSLATIONS);
 
+	#formFieldRef = inject(FORM_FIELD_INSTANCE);
+
+	formFieldSize = this.#formFieldRef.size();
```

Comment:
We need this to stay reactive !

```suggestion
	formFieldSize = this.#formFieldRef.size();
```